src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/utils/QueryDataAnonymizerTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.unittest.utils;
+
+import com.amazon.opendistroforelasticsearch.sql.utils.QueryDataAnonymizer;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class QueryDataAnonymizerTest {
+
+    @Test
+    public void queriesShouldHaveAnonymousFieldAndIndex() {
+        String query = "SELECT ABS(balance) FROM accounts WHERE age > 30 GROUP BY ABS(balance)";
+        String expectedQuery = "(\n" +
+                "\tSELECT ABS(identifier)\n" +
+                "\tFROM table\n" +
+                "\tWHERE identifier > number\n" +
+                "\tGROUP BY ABS(identifier)\n" +
+                "\t)";
+        Assert.assertEquals(expectedQuery, QueryDataAnonymizer.anonymizeData(query));
```

Comment:
Curious what will be the output in log4j, does it output multiple line for the same query?
If so, I would suggest to remove the newline to make it in same line. Because usually, we process the log data by using line based tools and group the data with request id. e.g., grep requestId=xxx which will return all the log corresponding to specific request.

Reply:
yes, it outputs multiple lines and tabs in the front of every line as well. ok that makes sense, i will remove the redundant returns and tabs.